### PR TITLE
Make only one package.

### PR DESCRIPTION
**Pull request by mookid**
This simplifies greatly working with the go command and gets rid of
the ugly references to the hosting location in the code.

### CODE REVIEW

File: peer.go [hunk 1]
```diff
@@ -200,9 +198,9 @@ func (p *peer) downloadPiece(piece *Piece) ([]byte, error) {
 	return res, nil
 }
 
-// Download creates a new peer that downloads pieces from a file
-func Download(handshake []byte, address string, pieces chan *Piece, results chan<- *Result) {
-	peer, err := new(handshake, address)
+// DownloadPieces creates a new peer that downloads pieces from a file
+func DownloadPieces(handshake []byte, address string, pieces chan *Piece, results chan<- *Result) {
+	peer, err := newPeer(handshake, address)
```
Comment (mookid) [inline peer.go:203]: this should probably be a method on peer, right?

Comment (matei-oltean) [reply peer.go:203]: It might, but the creation of the peer would have to be moved higher up. In another PR?

**Comment by matei-oltean** [comment]
Making everything a single package makes indeed things tidier. 
We might want to make all functions private? (except for main of course)

File: peer.go (hunk 1 above)
Comment (mookid) [reply peer.go:203]: sure.

**Review by matei-oltean — DISMISSED** [review]

**Comment by mookid** [comment]
> Making everything a single package makes indeed things tidier.
> We might want to make all functions private? (except for main of course)

I think it is too disruptive of a change for that PR.

**Review by matei-oltean — APPROVED** [review]

**Pull request merged**